health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/GenericTemplateScreen.js

```
@@ -72,23 +75,22 @@ const GenericTemplateScreen = ({ components = [], t, selectedField , templateNam
             right: 0,
             background: "#fff",
             padding: "1rem",
-            boxShadow: "0 -2px 4px rgba(0,0,0,0.1)",
             display: "flex",
             flexDirection: "column",
             gap: "0.5rem",
           }}
         >
           {buttonFields.map((field, index) => (
+            <div className={`${selectedField?.jsonPath === field.jsonPath ? "app-preview-field-pair app-preview-selected" : ""}`}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)